spec/unit/application/inspect_spec.rb

```diff
@@ -31,6 +31,12 @@
       Puppet[:report] = false
       expect { @inspect.setup }.to raise_error(/report=true/)
     end
+
+    it "should respect the catalog_cache_terminus if set" do
```

Comment:
Do you think it's worth it to check that the default is `:json`? Then we'd know it was broken again if we changed our default cached catalog storage format.